editors/emacs/lambdapi-vars.el

```diff
@@ -83,5 +86,11 @@
 Useful for people who prefer to have the dropdown window display systematically,
 since the window won't display if there are quail candidates.")
 
+;; Keybindings for goals display
+(global-set-key (kbd "C-c C-c") 'lp-display-goals)
+(global-set-key (kbd "C-c C-i") 'toggle-interactive-goals)
+(global-set-key (kbd "C-c C-p") 'lp-proof-backward)
+(global-set-key (kbd "C-c C-n") 'lp-proof-forward)
```

Comment:
Hmm finally I think it might be better to put this as well into `lambdapi-proofs`, to avoid adding dependencies on this file, sorry.